compute/admin_guide/authentication/oidc.adoc

```diff
@@ -116,12 +115,92 @@ Configure Prisma Cloud.
 
 .. Click *Save*.
 
+[.task]
+=== Azure Active Directory (AD)
+
+To integrate with Azure Active Directory (AD), you must register Prisma Cloud as an Open ID Connect (OIDC) application in Azure and configure Azure AD as an identity provider in Prisma Cloud.
+
+[.procedure]
+
+. Go to https://portal.azure.com/#home[your Azure console].
+
+. Find the Azure AD service.
+
+. Click the *app registration* button and select *New registration*
+
+. Enter a name and select *Twistlock only - single tenant* as the supported account type.
+
+. Under *Redirect URI* select *Web platform* and enter the path to the Prisma Cloud Console API for OIDC, for example https://`<CONSOLE_URL>`:8083/api/v1/authenticate/callback/oidc
```

Comment:
Web platform -> Rephrased by Azure to ‘Web'
CONSOLE_URL -> not consistent with previous https://<CONSOLE>:<PORT>/api/v1/authenticate/callback/oidc convention.

Reply:
@Tomerit Thank you for letting me know, but I don't understand from the comment what the new phrasings should be:
Should it be *Web console URL* instead of web platform?
Should it be `<WEB_CONSOLE_URL>` instead of `<CONSOLE_URL>`?

Reply:
https://<CONSOLE>:<PORT>/api/v1/authenticate/callback/oidc
Like in PingOne and Okta